### DISCUSSION TITLE
Using ESPAltherma with EmonHP / EmonCMS without Home Assistant.

### DISCUSSION THREAD

**Discussion by knightphoenix** [raomin/ESPAltherma#385] [Q&A]
I am looking to implement ESPAltherma directly with the Open Energy Monitor Level 3 system. The only issue is that the topic needs to start with the prefix emon/# instead of espaltherma/#.  Is it possible to update this without having the create a MQTT bridge to update all topics?

**Comment by sammort** [comment]
You can safely change `/espaltherma/OneATTR/` on [L57 of setup.h](https://github.com/raomin/ESPAltherma/blob/main/src/setup.h#L56-L57) to something like `/emon/ASHP/` and then your topics should appear in emoncms's inputs (your inputs will be grouped under a device name defined by the second string (so in my example the device name would be "ASHP")
